Dockerfile

```diff
@@ -36,13 +36,13 @@ COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-agent-port.groov
 
 # jenkins version being bundled in this docker image
 ARG JENKINS_VERSION
-ENV JENKINS_VERSION ${JENKINS_VERSION:-2.7.4}
+ENV JENKINS_VERSION ${JENKINS_VERSION:-2.19.2}
```

Comment:
Wondering if it makes sense we bring these default values (version and SHA) from the upstream OSS Dockerfile.

Shouldn't we just not use default values so that the build fails if no proper build-args defined?